exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/OkHttpGrpcExporter.java

```diff
@@ -212,7 +212,8 @@ public CompletableResultCode shutdown() {
     return CompletableResultCode.ofSuccess();
   }
 
-  static boolean isRetryable(Response response) {
+  /** Whether response is retriable or not. */
+  public static boolean isRetryable(Response response) {
```

Comment:
Is this change needed?

Reply:
yes, otherwise I would have to copy the whole method.

Reply:
copying it for now